lib/libutils/ext/include/asm.S

```diff
@@ -32,10 +32,13 @@
 	.balign \align
 	\name \colon
 UNWIND(	.fnstart)
+	.ifc	\attribute\(),naked
```

Comment:
```
.ifc \bti\(),default
BTI(bti c)
.endif
```
Same below

Reply:
Ok.. will change at all places.

Reply:
One issue with this change, assembled replaces bti in BTI(bti c) also with default resulting in assembly instruction
default c

So, we will have to search for a better name for this. attribute and it's value naked were more in sync with what gcc has as Jens suggested - https://gcc.gnu.org/onlinedocs/gcc/ARM-Function-Attributes.html

Reply:
> One issue with this change, assembled replaces bti in BTI(bti c) 

Oh, too bad :-/ maybe you can use `_bti` or `with_bti`. Using the term `naked` for assembly code is a bit weird because the assembler would never add any prologue/epilogue code anyway so what does this term really designate?
Anyways not a serious concern, please use something that makes sense to you as you read the code ;-)